test/lib/usd/translators/fixturesUtils.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from maya import cmds
+from maya import standalone
+
+import os
+import shutil
+
+_exportTranslatorName = "USD Export"
+
+def _setUpClass(modulePathName):
+    '''Common code for setUpClass() and readOnlySetUpClass()'''
+    standalone.initialize('usd')
+
+    cmds.loadPlugin('mayaUsdPlugin', quiet=True)
```

Comment:
Is it worth making the plugin load optional maybe? I'm thinking for export tests in particular, is it worth verifying that the `requires` statement in the Maya file we're loading is what causes the necessary plugin(s) to get loaded to ensure that we're not masking some kind of plugin load/dependency issue by loading manually?

We were admittedly somewhat sloppy about that in the original tests, but we eventually got a bit better about only explicitly loading when necessary.

Reply:
Done, see
https://github.com/Autodesk/maya-usd/pull/595/commits/5d1f55824fefc3319119ba94122f64e5b83844b6

Reply:
Cool, thanks!